Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.mm

```diff
@@ -134,11 +139,26 @@ - (NSString *)displayDescription
     return _webExtension->displayDescription();
 }
 
+- (NSString *)displayActionLabel
+{
+    return _webExtension->displayActionLabel();
+}
+
 - (NSString *)version
 {
     return _webExtension->version();
 }
 
+- (nullable WebKit::WebExtension::PlatformImage *)iconForSize:(CGSize)size
```

Comment:
We don't normally include nullability annotations in implementation files.

(5 other instances below)

Reply:
Oops, yep!